tools/manifest/sourcefile.py

```diff
@@ -29,6 +29,10 @@
                    WebDriverSpecTest)
 from .utils import cached_property
 
+# Cannot do `from ..metadata.webfeatures.schema import WEB_FEATURES_YML_FILENAME`
+# because relative import beyond toplevel throws *ImportError*!
+from metadata.webfeatures.schema import WEB_FEATURES_YML_FILENAME  # type: ignore
```

Comment:
Note for reviewers: This is similar to this section:

https://github.com/web-platform-tests/wpt/blob/7c6e57968932c49879182ae0fbbe1a949484ccee/tools/manifest/vcs.py#L12-L14